datasetinsights/estimators/deeplab.py

```diff
@@ -94,25 +94,24 @@ class DeeplabV3(Estimator):
         lr_scheduler: pytorch learning rate scheduler
     """
 
-    def __init__(self, *, config, writer, checkpointer, device, **kwargs):
+    def __init__(self, config, writer, checkpointer, device, **kwargs):
         self.config = config
+        self.writer = writer
+        self.checkpointer = checkpointer
+        self.device = device
 
-        self.backbone = config.backbone
-        self.num_classes = config.num_classes
+        self.backbone = self.config.backbone
```

Comment:
I don't see any difference between the two, but i can change it if its an issue

Reply:
In theory it will save some "lookup" time as `self.xxx` requires reference to the class object. This is very minor so it's not very important.... that's why I put a `[nit]` prefix here.

Reply:
Will change it